src/ontology/uberon-edit.obo

```diff
@@ -66003,7 +66004,12 @@ xref: SCTID:369083001
 xref: UMLS:C0228271 {source="BIRNLEX:983"}
 xref: Wikipedia:Uncinate_fasciculus
 is_a: UBERON:0005838 {source="FMA"} ! fasciculus of brain
-relationship: part_of UBERON:0003023 {source="NIFSTD"} ! pontine tegmentum
+relationship: overlaps UBERON:0002153 ! fastigial nucleus
+relationship: overlaps UBERON:0002275 ! reticular formation
+relationship: overlaps UBERON:0002298 ! brainstem
+relationship: overlaps UBERON:0002726 ! cervical spinal cord
+relationship: overlaps UBERON:0003023 {source="NIFSTD"} ! pontine tegmentum
```

Comment:
@shawntanzk I'd remove the NIFSTD source, because it's not the source of this overlaps rel.

Reply:
That was there on the part_of relation from before - I wonder if it shows provenance? cause the definition doesn't mention pons/pontine tegmentum at all. Will try to look into NIFSTD to check this, if not find why this is there from literature

Reply:
Might be worth keeping as NIFSTD is at least the source of a partonomy relationship, even if they didn't choose the right one.

Reply:
ok trying to open NIF on protege is killing it lol, im going to assume that the partonomy relationship is there (don't see partonomy relationships in general on bioportal where the user facing nif is at) - either way, since it was there before I think it should be kept